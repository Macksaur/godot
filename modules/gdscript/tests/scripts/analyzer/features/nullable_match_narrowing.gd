func test():
    var nullable_string: String? = null
    match nullable_string:
        "123", "13":
            print(nullable_string.begins_with("1"))
        {}:
            print(nullable_string.begins_with("1"))
        []:
            print(nullable_string.begins_with("1"))
        null:
            prints("Nullable is null! Proof: ", nullable_string)
        var nullable:
            prints("Nullable is null! Proof: ", nullable)