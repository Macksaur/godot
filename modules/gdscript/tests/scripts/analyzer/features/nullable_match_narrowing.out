GDTEST_OK
Nullable is null! Proof:  <null>
